crates/orderbook/src/database/orders.rs

```diff
@@ -221,18 +224,20 @@ async fn insert_order(order: &Order, ex: &mut PgConnection) -> Result<(), Insert
 }
 
 async fn insert_quote(
-    uid: &OrderUid,
+    order: &Order,
     quote: &Quote,
     ex: &mut PgConnection,
 ) -> Result<(), InsertionError> {
     let quote = database::orders::Quote {
-        order_uid: ByteArray(uid.0),
+        order_uid: ByteArray(order.metadata.uid.0),
         gas_amount: quote.data.fee_parameters.gas_amount,
         gas_price: quote.data.fee_parameters.gas_price,
         sell_token_price: quote.data.fee_parameters.sell_token_price,
         sell_amount: u256_to_big_decimal(&quote.sell_amount),
         buy_amount: u256_to_big_decimal(&quote.buy_amount),
         solver: ByteArray(quote.data.solver.0),
+        call_data: order.metadata.full_app_data.clone().unwrap_or_default(),
```

Comment:
Appdata is meta data that a user can attach to their order while the calldata is the function call data a solver proposed with their quote.

Reply:
I've updated passing the `call_data` from response to the /quote request to the database.